easybuild/easyconfigs/s/SPLITREAD/SPLITREAD-0.1-foss-2016b.eb

```diff
@@ -0,0 +1,66 @@
+#  easybuild easyconfig
+#
+#  John Dey  jfdey@fredhutch.org
+#
+#  Fred Hutchinson Cancer Research Center
+#
+#  Notes: SPLITREAD distribution does not have a Makefile. Splitread is build by running a single shell script with
+#  no parameters called:  Make.sh.  
+#  
+#  Make.sh installs 'copies' to <$HOME>.
+#  Use prebuildopts 'parameter' to define $HOME and to execute Make.sh
+# 
+
+easyblock = 'MakeCp'
+
+name = 'SPLITREAD'
+version = '0.1'
+
+homepage = 'http://splitread.sourceforge.net/'
+description = """SPLITREAD is a novel method for detecting INDELs (small insertions and deletion with size less than 50bp) as well as large deletions that are within the coding regions from the exome sequencing data. It also can be applied ot the whole genome sequencing data."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = [('http://sourceforge.net/projects/%(name)s/files/', 'download')]
+sources = ['%(name)s_v%(version)s.tar.gz']
+
+start_dir = 'Code-Release'
+
+prebuildopts = ' mkdir %(installdir)s/bin && '
+prebuildopts += 'HOME=%(installdir)s/bin && ' 
+prebuildopts += '/bin/bash Make.sh # '
```

Comment:
this looks... hacky (especially the `#` part)

Maybe this deserves a dedicated easyblock?